packages/codegen-ui-react/lib/forms/form-renderer-helper.ts

```diff
@@ -421,6 +422,57 @@ export const addFormAttributes = (component: StudioComponent | StudioComponentCh
   return attributes;
 };
 
+/**
+  if (errors.name?.hasError) {
+    await runValidationTasks("name", value);
+  }
+ */
```

Comment:
Was this comment left intentionaly?

Reply:
Yes, those kind of comments are basically representations of what the ast writer will output